packages/discord.js/typings/index.d.ts

```diff
@@ -1776,12 +1776,13 @@ export class ReactionEmoji extends Emoji {
 
 export class RichPresenceAssets {
   private constructor(activity: Activity, assets: RawRichPresenceAssets);
-  public largeImage: Snowflake | null;
+  public largeImage: string | null;
   public largeText: string | null;
-  public smallImage: Snowflake | null;
+  public smallImage: string | null;
```

Comment:
kinda redundant, I know, but this is done elsewhere and if the Snowflake type is ever changed it can be changed here too
```suggestion
  public smallImage: Snowflake | string | null;
```

Reply:
I mean Snowflake | string resolves to just `string`

Reply:
Yeah I know but

> this is done elsewhere and if the Snowflake type is ever changed it can be changed here too